42mad

```diff
@@ -15,6 +15,7 @@ url_gapps="https://madatv.b-cdn.net/open_gapps-arm64-7.1-pico-20200715.zip"
 rgcconf="/data/data/de.grennith.rgc.remotegpscontroller/shared_prefs/de.grennith.rgc.remotegpscontroller_preferences.xml"
 pdconf="/data/data/com.mad.pogodroid/shared_prefs/com.mad.pogodroid_preferences.xml"
 useragent='Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:53.0) Gecko/20100101 Firefox/53.0'
+ip="$(ifconfig | grep -A 1 'eth0' | tail -1 | cut -d ':' -f 2 | cut -d ' ' -f 1)"
```

Comment:
your command:
kvim2:/ $ ifconfig | grep -A 1 'eth0' | tail -1 | cut -d ':' -f 2 | cut -d ' ' -f 1                                                                                                                                                          
Ethernet
192.168.1.19


My suggestion:
kvim2:/ $ ifconfig|grep -A 1 'eth0'|awk '/inet addr/{print $2}'|cut -d ':' -f 2                                                                                                                                                              
192.168.1.19

Reply:
oh and for the record, your command could be fixed by using tail -n 1 instead of tail -1. gnu tail in linux spoiled you :)

Reply:
> your command:
> kvim2:/ $ ifconfig | grep -A 1 'eth0' | tail -1 | cut -d ':' -f 2 | cut -d ' ' -f 1
> Ethernet
> 192.168.1.19
> 
> My suggestion:
> kvim2:/ $ ifconfig 'eth0'|awk '/inet addr/{print $2}'|cut -d ':' -f 2
> 192.168.1.19

Agreed and changed. In the end IP address is all that matters.